http/client.go

```diff
@@ -212,6 +212,7 @@ func (c *Client) BinaryRequest(method, url, token string, reqData *RequestData,
 	if err != nil {
 		return
 	}
+	reqData.RespValue = resp.StatusCode
```

Comment:
This seems to me to be rather an abuse of the RespValue field, perhaps add a new field to the RespStatus if that is required.